### PR TITLE
Feature/uar 591 update json attributes

### PR DESCRIPTION
**Pull request by AndreKramer7654**
### JIRA link

https://companieshouse.atlassian.net/browse/UAR-591

### Change description

Add attributes on JSON elements related to changes.

Fix
isOnSanctionsList to be Boolean so it can be optional

Fix
address mappings to use line2 and town.

Rename 
corporateSoleName to corporateName as it's always that in the XML.

Change
Make DueDiligence use Address instead of AddressDto

Change to use BenificialOwnerStatementType in update JSON

renamed Person.name to Person.fullName

### Work checklist

- [ ] Tests added where applicable

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by bantao-ch — APPROVED** [review]

**Review by azafarCH — APPROVED** [review]
changes for dueDiligence look good 👍

**Review by bantao-ch — CHANGES_REQUESTED** [review]
Need a change to Presenter attribute -- name to "full_name"

**Comment by AndreKramer7654** [comment]
> Need a change to Presenter attribute -- name to "full_name"

Done but fullName as JSON is camel cased elesewhere

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/Presenter.java [hunk 1]
```diff
@@ -3,18 +3,18 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 public class Presenter {
-    @JsonProperty("name")
-    private String name;
+    @JsonProperty("fullName")
```
[the remark this answers is not in the document]
Comment (bantao-ch) [reply src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/Presenter.java:6]: As discussed we will instead create a new Presenter class to match in CHIPS Filing Consumer

**Review by bantao-ch — APPROVED** [review]

**Review by jkayyala — APPROVED** [review]

**Pull request merged**